pkg/urbit/vere/disk.c

```diff
@@ -669,7 +669,7 @@ u3_disk_info(u3_disk* log_u)
   while ( red_u ) {
     lit = u3nc(
       u3_pier_mass(
-        c3__read,
+        u3dt("cat", 3, c3__read, u3dc("scot", c3__ux, u3i_chub((c3_d)red_u))),
```

Comment:
I don't think this kind of naming convention will be useful to consumers. But I expect we'll need to iterate on this interface quite a bit. The module-specific `%info` implementations will also have to change quite a bit due to the mars/urth architecture. So these should be considered first drafts. (Nothing to do here, just noting ...)

Reply:
yeah, agreed. no obvious way to change this while preserving the constraint that flattened variable names are unique, so i'll leave it as-is for now.

Reply:
(another option would have been to split these out into multiple levels - `c3__read` at top level, and the address below. that would mildly help clients who were depending on the hierarchical structure, but not clients who were depending on the flattened variable name. can do it if desired.)

Reply:
(went ahead and did this)